### PR TITLE
Feature/smar-125-127

### PR DESCRIPTION
**Pull request by RaznoOleg**
## Description changes:
• Connected frontend to backend for appointments scheduler.

### General
- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Don't have "any" on my code
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside ternary operator
- [x] Don't have commented code
- [x] All links in on ENV files
- [x] Used camelCase for variables and functions
- [x] Functions is public only if it's used outside the class
- [x] No hardcoded values
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend
- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

#### General view of the appointment scheduler:
image

#### Clicked on a certain day and get a list of appointments for that day - if we log in to the system as a local doctor, then the last name of the remote doctor will be displayed on the card with information about the event, on the other hand, if we log in as a remote doctor, then the last name of the local doctor will be displayed:
image

#### Clicked on a certain event and we got information only about this event:
image

#### Clicked on a day when there are no events and we received a corresponding message:
image

#### General view of the weekly appointment scheduler:
image

#### General view of the daily appointment scheduler:
image

#### General view of the "Agenda":
image

#### Clicked in the "Agenda" on a certain event and got information only about this event:
image

#### Clicking on the patient's name opens his card:
image
image

### CODE REVIEW

File: src/components/AppointmentsScheduler/AppointmentData/index.tsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  const patientAge = `${
+    new Date().getFullYear() - new Date(patient?.birthDate).getFullYear()
+  } ${t('Appointments.years')}`;
+
+  const patientFullName = `${patient.firstName.charAt(0)}. ${patient.lastName}`;
+
+  const patientGenderAge = `${patient.gender}, ${patientAge}`;
+
+  const doctorRole =
+    doctor.role === 'Local'
```
Comment (lyudad) [inline src/components/AppointmentsScheduler/AppointmentData/index.tsx]: we dont compare with strings

File: src/components/AppointmentsScheduler/AppointmentData/index.tsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      ? t('Appointments.remoteDoctor')
+      : t('Appointments.localDoctor');
+
+  const doctorLastName = `${t('Appointments.doctor')} ${
```
Comment (lyudad) [inline src/components/AppointmentsScheduler/AppointmentData/index.tsx:51]: same

File: src/components/AppointmentsScheduler/AppointmentData/index.tsx (hunk 1 above)
Comment (RaznoOleg) [reply src/components/AppointmentsScheduler/AppointmentData/index.tsx]: fixed

File: src/components/AppointmentsScheduler/AppointmentData/index.tsx (hunk 2 above)
Comment (RaznoOleg) [reply src/components/AppointmentsScheduler/AppointmentData/index.tsx:51]: fixed

**Pull request merged**